operations/commit_queue.go

```diff
@@ -217,17 +218,24 @@ func enqueuePatch() cli.Command {
 	return cli.Command{
 		Name:  "enqueue-patch",
 		Usage: "enqueue an existing patch on the commit queue",
-		Flags: mergeFlagSlices(addYesFlag(), addPatchIDFlag(cli.BoolFlag{
-			Name:  forceFlagName,
-			Usage: "force item to front of queue",
-		})),
+		Flags: mergeFlagSlices(addYesFlag(), addPatchIDFlag(
+			cli.BoolFlag{
+				Name:  forceFlagName,
+				Usage: "force item to front of queue",
+			},
+			cli.StringFlag{
+				Name:  commitMessageFlag,
+				Usage: "commit message for the new commit (default is the existing patch description)",
```

Comment:
Do we now always use the default patch description?

Reply:
before these changes we use whatever the existing patch's description is as the commit message